test/e2e/message-types.test.ts

```
@@ -0,0 +1,407 @@
+            firstName: 'Test',
+            userId: 123456,
+            username: 'testuser',
+        });
+
+        // Add voice to the message
+        voiceMessage.message!.voice = {
+            duration: 10,
+            file_id: 'voice_file_id',
+            file_unique_id: 'unique_voice_id',
+        };
+        delete voiceMessage.message!.text;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)